bluetooth/README.md

```diff
@@ -0,0 +1,41 @@
+## Permissions
+
+This library provide support for out-of-the-box access to Bluetooth.
+
+
+### Usage
+Create a `Bluetooth` object through the `BluetoothBuilder`. This gives you access to a `Flow` of Bluetooth devices. To scan for devices simply call
```

Comment:
I think it's good to place code snipped with `Bluetooth` creation with details what is `ConnectionSettings`.

Reply:
Let's do this in the refresh of the API.